dataactbroker/handlers/domainHandler.py

```diff
@@ -0,0 +1,28 @@
+from dataactcore.utils.jsonResponse import JsonResponse
+from dataactcore.utils.statusCode import StatusCode
+
+
+class DomainHandler:
+
+    def __init__(self, interfaces=None):
+        if interfaces is not None:
+            self.interfaces = interfaces
+            self.validationManager = interfaces.validationDb
+
+    def addInterfaces(self, interfaces):
+        """ Add connections to databases
+
+        Args:
+            interfaces: InterfaceHolder object to DBs
+        """
+        self.interfaces = interfaces
+        self.validationManager = interfaces.validationDb
+
+    def listAgencies(self):
+        agencies = self.validationManager.getAllAgencies()
+        agency_list = []
+
+        for agency in agencies:
```

Comment:
(though I realize you all mocked this up already...am more curious about getting the overall surrogate key question answered)